src/components/Breadcrumb/Breadcrumb.json

```diff
@@ -1,6 +1,7 @@
 {
   "name": "Breadcrumb",
   "notes": "Shows the user's current location in a hierarchy and provides a means of navigating upward.",
+  "description": "Shows the user's current location in a hierarchy and provides a means of navigating upward.",
```

Comment:
I would change to: "Shows the user's current location in a hierarchy and provides a means of navigating up or down."